analysis/topeft_run2/run_analysis.py

```diff
@@ -32,85 +32,174 @@
     "nSumOfWeights_renormfactDown",
 ]
 
-if __name__ == '__main__':
-
-    parser = argparse.ArgumentParser(description='You can customize your run')
```

Comment:
I do like this older style, but I guess the newer style ensures every line fits on the screen?